design/006-renku-api-activities-plans/006-renku-api-activities-plans.md

```diff
@@ -203,7 +203,7 @@ all_activities = Activity.list()
 activities_by_input = Activity.filter_by_input(path="data/my.csv")
 activities_by_input_parent = Activity.filter_by_input(path="data/", exact=False)
 activities_by_output = Activity.filter_by_output(path="data/my.csv")
-activities_by_output_parent = Activity.filter_by_output(path="data/", with_children=True)
+activities_by_output_parent = Activity.filter_by_output(path="data/", exact=True)
```

Comment:
sorry, I'm confused by this change. `with_children=True` and `exact=True` seem to be doing the opposite... Or maybe I don't understand what `exact` is supposed to be doing?

Reply:
my bad... I blame monday mornings.